package.json

```diff
@@ -12,22 +12,22 @@
     "Mihai Bazon <mihai.bazon@gmail.com> (http://lisperator.net/)"
   ],
   "repository": "mishoo/UglifyJS",
-  "main": "tools/node.js",
+  "main": "index.js",
   "bin": {
     "uglifyjs": "bin/uglifyjs"
   },
   "files": [
-    "bin",
-    "lib",
-    "tools",
+    "index.js",
```

Comment:
What's the point of generating the source map if we don't include it in the package? 👻

Reply:
... also, don't drop `bin` as our CLI script is in there.